packages/lpa-questionnaire-web-app/patches/connect-mongodb-session+2.4.1.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/node_modules/connect-mongodb-session/index.js b/node_modules/connect-mongodb-session/index.js

```

Comment:
Does the LPA Questionnaire need sessions? I suspect it does, but if not then we don't need this patch

Reply:
I believe it does, but can confirm that tomorrow